machine/wnc_x86_64/wnc.patch/installer/x86_64/grub.d/50_onie_grub

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
Same question:

What is this file for?  You should not have to patch this file.

What is the default 50_onie_grub lacking that you require?